esphome/components/audio_dac/__init__.py

```diff
@@ -0,0 +1,59 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import automation
+from esphome.automation import maybe_simple_id
+from esphome.core import coroutine_with_priority
+from esphome.const import CONF_ID, CONF_VOLUME
+
+CODEOWNERS = ["@kbx81"]
+IS_PLATFORM_COMPONENT = True
+
+audio_dac_ns = cg.esphome_ns.namespace("audio_dac")
+AudioDac = audio_dac_ns.class_("AudioDac")
+
+MuteOffAction = audio_dac_ns.class_("MuteOffAction", automation.Action)
+MuteOnAction = audio_dac_ns.class_("MuteOnAction", automation.Action)
+SetVolumeAction = audio_dac_ns.class_("SetVolumeAction", automation.Action)
+
+
+MUTE_ACTION_SCHEMA = maybe_simple_id(
+    {
+        cv.GenerateID(): cv.use_id(AudioDac),
+    }
+)
+
+SET_VOLUME_ACTION_SCHEMA = cv.maybe_simple_value(
+    {
+        cv.GenerateID(): cv.use_id(AudioDac),
+        cv.Required(CONF_VOLUME): cv.templatable(
+            cv.All(cv.decibel, cv.float_range(max=24, min=-63.5))
```

Comment:
Should this be a 0-1 float / percentage and the dac platform itself should convert to the correct range?

Reply:
Yes indeed -- makes more sense now. Done in [`e0c269d` (#59)](https://github.com/esphome/voice-kit/pull/59/commits/e0c269df0fdf72ed772a8b2230ff2543f2cda301)